test/e2e/functional/tests/e2e.go

```diff
@@ -239,8 +239,9 @@ var _ = Describe("metallb", func() {
 					AutoAssign: &autoAssign,
 					BGPAdvertisements: []metallbv1alpha1.BgpAdvertisement{
 						{
-							AggregationLength: 24,
-							LocalPref:         100,
+							AggregationLength:   pointer.Int32Ptr(24),
```

Comment:
Can you add another entry with no aggregation lenght to see if the defaulting works?